Quasardb/QdbCluster.cs

```diff
@@ -109,12 +109,15 @@ public override bool IsInvalid
         /// Gets the last API error description.
         /// <returns>A message describing the error occurred during the last operation.</returns>
         /// </summary>
-        public string GetLastError()
+        public unsafe string GetLastError()
         {
             qdb_error error;
-            qdb_sized_string message;
-            qdb_api.qdb_get_last_error(_handle, out error, out message);
-            return message.ToString();
+            qdb_sized_string* message = (qdb_sized_string*)IntPtr.Zero;
+            var ec = qdb_api.qdb_get_last_error(_handle, out error, out message);
+            QdbExceptionThrower.ThrowIfNeeded(ec);
```

Comment:
Is this necessary? I would *assume( get GetLastError() is literally just a function to get the last error message, but it appears here it also throws. Also, doesn't QdbExceptionThrower.ThrowIfNeeded() *also* call get_last_error() 

It seems like a bit weird logic to me, but maybe I'm misunderstanding, what is the intention here?

Reply:
"ThrowIfNeeded() also call get_last_error()" - no, doesn't call this, 
it just generates an own message.
There is another stuff for this ThrowIfNeededWithMsg()

Reply:
And as I can see the exception throwing inside this function must not be a problem.

Reply:
Ah, I see, was just trying to understand it. I trust it works well. 

Then by all means, go ahead and merge.